prod_line_ctrl:
  ros__parameters:
    hkclr_ip: 127.0.0.1
    hkclr_port: 8000
    jinli_ip: 127.0.0.1
    jinli_port: 8080

/**/dis_sta_node:
  ros__parameters:
    ip_start: 192.168.8.100
    port: "4840"
    simulation: False